src/ray/core_worker/lib/java/io_ray_runtime_object_NativeObjectStore.cc

```diff
@@ -16,10 +16,15 @@
 
 #include <jni.h>
 
-#include "jni_utils.h"
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
 #include "ray/common/id.h"
 #include "ray/core_worker/common.h"
 #include "ray/core_worker/core_worker.h"
+#include "jni_utils.h"  // NOLINT(build/include_subdir)
```

Comment:
Curious did cpplint make this change?

Reply:
Since generally header inclusions happen in alphabetical order

Reply:
My mistake. I reordered the includes during the trial-and-error process.  
I have now restored the original include position and applied the formatting script.